package.json

```diff
@@ -18,7 +19,9 @@
   },
   "jest": {
     "scriptPreprocessor": "preprocessor.js",
-    "unmockedModulePathPatterns": ["node_modules/react"]
```

Comment:
we need to add a `prepublish` script to the `"scripts"` property that builds the JSX